python/pip_install/pip_repository_requirements.bzl.tmpl

```diff
@@ -18,28 +18,34 @@ def _clean_name(name):
     return name.replace("-", "_").replace(".", "_").lower()
 
 def requirement(name):
-    return "@%%NAME%%_" + _clean_name(name) + "//:pkg"
+    return "%%MACRO_TMPL%%".format(_clean_name(name), "pkg")
 
 def whl_requirement(name):
-    return "@%%NAME%%_" + _clean_name(name) + "//:whl"
+    return "%%MACRO_TMPL%%".format(_clean_name(name), "whl")
 
 def data_requirement(name):
-    return "@%%NAME%%_" + _clean_name(name) + "//:data"
+    return "%%MACRO_TMPL%%".format(_clean_name(name), "data")
 
 def dist_info_requirement(name):
-    return "@%%NAME%%_" + _clean_name(name) + "//:dist_info"
+    return "%%MACRO_TMPL%%".format(_clean_name(name), "dist_info")
 
 def entry_point(pkg, script = None):
     if not script:
         script = pkg
-    return "@%%NAME%%_" + _clean_name(pkg) + "//:rules_python_wheel_entry_point_" + script
+
+    ep = "@%%NAME%%_{}//:rules_python_wheel_entry_point_{}".format(_clean_name(pkg), script)
+    if str(Label("//:unused")).startswith("@@"):
+        ep = "@" + ep
+
+    return ep
 
 def _get_annotation(requirement):
     # This expects to parse `setuptools==58.2.0     --hash=sha256:2551203ae6955b9876741a26ab3e767bb3242dafe86a32a749ea0d78b6792f11`
     # down to `setuptools`.
     name = requirement.split(" ")[0].split("=")[0].split("[")[0]
     return _annotations.get(name)
 
+# TODO name needs to change
```

Comment:
Could you elaborate here, please?

Reply:
I have it working now.  It was a note while I migrated.